.github/workflows/tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         os: ['ubuntu-latest']
         python-version: [
           '3.9',
```

Comment:
note: let's raise this up to 3.10 at some point in the **near ** future

Reply:
Alternatively, we can keep it until GHA phases it out?